### PR TITLE
Add plugins, stores and controller view doc

### PR DESCRIPTION
**Pull request by redonkulus**
@jedireza @Vijar

### CODE REVIEW

File: docs/guides/plugins.md [hunk 1]
```diff
@@ -1,3 +1,67 @@
...
+var context = app.createContext({
+    foo: 'bar'
+});
+
+context.getComponentContext().getFoo(); // returns 'bar'
+// or this.props.context.getFoo() from a React component
+```
+
+Example plugins:
+ * [fluxible-plugin-fetchr](https://github.com/yahoo/fluxible-plugin-fetchr) - Polymorphic RESTful services
```
Comment (Vijar) [inline docs/guides/plugins.md]: What is this "polymorphic" business?

Comment (redonkulus) [inline docs/guides/plugins.md]: I copied it

Comment (redonkulus) [inline docs/guides/plugins.md]: @Vijar fixed

**Comment by Vijar** [comment]
:ship:

**Comment by jedireza** [comment]
:+1:

**Pull request merged**